### DISCUSSION TITLE
Questions about Mixed Queries

### DISCUSSION THREAD

**Discussion by meton-robean** [milvus-io/milvus#3363] [Q&A and General discussion] [answered]
There is a scene that requires a label on the vector. When searching, only the vectors with a certain label are compared. This is a small-scale comparison scene. Does version 0.12.0 have this function? How do you achieve this small-scale comparison by Label?

**Comment by miia12** [comment]
We will roll out the hybrid query feature in version 0.11.0. Please refer to the documentation and code for the detailed implementation :)

**Comment by miia12** [comment, marked as answer]
Please refer to Support DSL interface of [DESIGN.md](https://github.com/milvus-io/milvus/blob/master/DESIGN.md).
